mxnet/tests/unittests/test_import.py

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Should CI also run these individual `test_import()` calls with just the minimal sub-package installed?  Would that have caught the issue with importing only autogluon.mxnet without autogluon.extra being installed?

Without such an individual test per sub-package, there might again be unintended dependencies re-introduced between sub-packages in future commits?

Reply:
CI runs unit tests with minimal install (mostly, tabular installs text), so this should catch it.